MonsterWaves/Core/Game.hpp

```diff
@@ -1,6 +1,6 @@
 #pragma once
-#include "Window.hpp"
-#include "TimeStep.hpp"
+#include "Screen/Window.hpp"
+#include "Utils/TimeStep.hpp"
 #include <cmath>
```

Comment:
@JakubMastalski sugeruję rozdzielać includy twoich bibliotek i bibliotek standardowych ale to na jakimś odzielnym branchu to popraw i wystaw na mnie PR

Reply:
Przy okazji przypomina, że w main.cpp jest niepotrzebny include!